app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd, old_file, new_file = command.split()
+    if old_file == new_file:
+        return
```

Comment:
Why not simply using reverse condition here. That will exclude using empty return